### DISCUSSION TITLE
Where did the building noises go?

### DISCUSSION THREAD

**Discussion by coastercrab** [OpenRCT2/OpenRCT2#17516] [Feature requests]
Starting up the original games, I notice I miss the thud of building a track piece, raising/lowering land, placing objects etc.
I can't hear it in openrct2. I suspect some other sounds are missing/really quiet too.

Thank the heavens we're no longer being blasted with "now that's what I call the same 120db toilet sound" on repeat though!

**Comment by spacek531** [comment]
what are your volume settings?

![image](https://user-images.githubusercontent.com/3454156/177661952-36f4e750-2460-4912-9a26-75896711747d.png)

**Reply by coastercrab** [reply]
How do I delete a discussion? I think they're just quieter than I remember. They are definitely there oops.

**Reply by spacek531** [reply]
you can mark the discussion as answered
